### ISSUE TITLE
Model accurate missing after converting to tflite

### ISSUE THREAD

**Issue by Ajith61** [tensorflow/tensorflow#54980]
### 1. System information

- Windows 10
- Pip package
- 2.7.0

### 2. Code

import tensorflow as tf

model = tf.keras.models.load_model('liveness.model')
converter = tf.lite.TFLiteConverter.from_keras_model(model)
tflite_model = converter.convert()
open("liveness4.tflite", "wb").write(tflite_model)

### 3. Failure after conversion
If the conversion is successful, but the generated model is wrong, then state what is wrong:

- Model produces wrong results and/or has lesser accuracy.

Also Im  getting this warning after converting to tflite model.

WARNING:absl:Buffer deduplication procedure will be skipped when flatbuffer library is not properly loaded

**Comment by sushreebarsa** [comment]
@Ajith61 
In order to expedite the trouble-shooting process here,Could you please fill the issue [template](https://github.com/tensorflow/tensorflow/issues/new/choose),
Thanks!
